hw3/01-routing.js

```diff
@@ -1,31 +1,29 @@
-const http = require('http');
-const port = process.env.PORT || 5000;
+const http = require(`http`);
+const port = process.env.PORT || 5001;
 
-// http://localhost:5000/welcome should return a status code 200 with a welcome message of your choice in html format
 
-// http://localhost:5000/redirect should redirect the request to '/redirected' by using 302 as the status code / the redirected page should return a redirected message of your choice
+// http://localhost:5001/welcome should return a status code 200 with a welcome message of your choice in html format
+// http://localhost:5001/redirect should redirect the request to `/redirected` by using 302 as the status code / the redirected page should return a redirected message of your choice
+// http://localhost:5001/cache should return `this resource was cached` in html format and set the cache max age to a day
+// http://localhost:5001/cookie should return `cookies… yummm` in plain text and set `hello=world` as a cookie
+// http://localhost:5001/check-cookies should return `yes` / `no` in plain text depending on whether the browser has the `hello` cookie
+// For other routes, such as http://localhost:5001/other, this exercise should return a status code 404 with `404 - page not found` in html format
 
-// http://localhost:5000/cache should return 'this resource was cached' in html format and set the cache max age to a day
-
-// http://localhost:5000/cookie should return 'cookies… yummm' in plain text and set 'hello=world' as a cookie
-
-// http://localhost:5000/check-cookies should return 'yes' / 'no' in plain text depending on whether the browser has the 'hello' cookie
-
-// For other routes, such as http://localhost:5000/other, this exercise should return a status code 404 with '404 - page not found' in html format
 
 const server = http.createServer((req, res) => {
   const routes = [
-    'welcome',
-    'redirect',
-    'redirected',
-    'cache',
-    'cookie',
-    'check-cookies',
-    'other',
+    `welcome`,
+    `redirect`,
+    `redirected`,
+    `cache`,
+    `cookie`,
+    `check-cookies`,
+    `other`,
   ];
 
+
   let getRoutes = () => {
-    let result = '';
+    let result = ``;
```

Comment:
I think back ticks formatting is not required for strings

Reply:
That's correct; the back ticks are me just trying to get into the habit of using them everywhere.